src/models/layerMetadata/layer3DMetadata.ts

```diff
@@ -148,7 +149,7 @@ export class Layer3DMetadata implements ILayer3DMetadata, IMetadataCommonModel {
     isAutoGenerated: true,
   })
   //#endregion
-  public productVersion: string | undefined = undefined;
+  public productVersion: number | undefined = undefined;
```

Comment:
@alebinson wouldn't it cause a a conflict with our data (string 1.0.0)?

Reply:
Yes it will have an impact with APP only, want to minimize code changes in 3D